src/s3/explorer/s3BucketNode.ts

```diff
@@ -71,7 +70,7 @@ export class S3BucketNode extends AWSTreeNodeBase implements AWSResourceNode, Lo
         this.childLoader.clearChildren()
     }
 
-    private async loadPage(continuationToken: string | undefined): Promise<ChildNodePage> {
+    private async loadPage(continuationToken: string | undefined): Promise<ChildNodePage<S3FolderNode | S3FileNode>> {
```

Comment:
The return type annotation isn't explicitly necessary but can be nice to have sometimes.